src/SDKs/CognitiveServices/dataPlane/Personalizer/Personalizer/Customizations/PersonalizerClientExtensions.cs

```diff
@@ -0,0 +1,89 @@
+using Microsoft.Azure.CognitiveServices.Personalizer.Models;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
I think we should add a copyright header to these files.

Reply:
fixed